packages/katello/rubygem-pulp_deb_client/rubygem-pulp_deb_client.spec

```diff
@@ -0,0 +1,96 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name pulp_deb_client
+%global gem_require_name %{gem_name}
+
+%global release 1
+%global prerelease b1
+%global prereleaserpm %{?prerelease:.}%{?prerelease}
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.3.0
+
+Release: %{?prereleaserpm:0.}%{release}%{?prereleaserpm}%{?dist}
+Summary: Pulp 3 DEB plugin API Ruby Gem
+Group: Development/Languages
+License: GPL-2.0+
+URL: https://github.com/pulp/pulp_deb/
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby >= 1.9
+Requires: %{?scl_prefix_ruby}ruby(rubygems) > 1.3.1
+Requires: %{?scl_prefix}rubygem(faraday) >= 0.14.0
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1.0
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1
+Requires: %{?scl_prefix_ruby}rubygem(json) < 3
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby >= 1.9
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel > 1.3.1
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
+
+%description
+Client bindings for the pulp_deb pulp3 plugin
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
```

Comment:
you'll need prerelease here to:  

%setup -q -D -T -n  %{gem_name}-%{version}%{?prerelease}